src/strategies/snet-liquidity-providers/index.ts

```diff
@@ -0,0 +1,76 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller, call } from '../../utils';
+
+export const author = 'bonustrack';
+export const version = '0.1.1';
```

Comment:
```suggestion
export const version = '0.1.0';
```

Reply:
fixed in https://github.com/snapshot-labs/snapshot-strategies/pull/202/commits/7bb1374abb5c34f45df2a839890fafac26c14253